apps/etl/extraction/sources/ifrc_event/extract.py

```diff
@@ -76,22 +75,33 @@ def store_extraction_data(  # type: ignore[reportIncompatibleMethodOverride]
         return resp_data
 
     @classmethod
-    def handle_extraction(cls, url: str, params: IFRCEventQueryVars, headers: dict, source: int) -> int:  # type: ignore[reportIncompatibleMethodOverride]
+    def handle_extraction(cls, metadata: IfrcEventExtractionInputMetadata) -> int:  # type: ignore[reportIncompatibleMethodOverride]
         """
         Process data extraction.
         Returns:
             int: ID of the extraction instance
         """
         logger.info("Starting data extraction")
 
-        instance = cls._create_extraction_instance(url=url, source=source)
+        url = f"{etl_config.IFRC_DATA_URL}/api/v2/event/?appeal_type=0,1"
```

Comment:
I think appeal_type can be added into metadata.

Reply:
If it's a variable, we can add them to appeal.